### ISSUE TITLE
Loss Divergence 

### ISSUE THREAD

**Issue by fentuoli** [thu-ml/tianshou#425]
- [ ] I have marked all applicable categories:
    + [x] exception-raising bug
    + [x] RL algorithm bug
    + [x] documentation request (i.e. "X is missing from the documentation.")
    + [x] new feature request
- [ ] I have visited the [source website](https://github.com/thu-ml/tianshou/)
- [ ] I have searched through the [issue tracker](https://github.com/thu-ml/tianshou/issues) for duplicates
- [ ] I have mentioned version numbers, operating system and environment, where applicable:
  ```python
  import tianshou, torch, numpy, sys
  print(tianshou.__version__, torch.__version__, numpy.__version__, sys.version, sys.platform)
  ```
During training, I found that the value of loss is really large, is this normal? I also found that my reward gradually decreased from fluctuations, and finally stabilized at a relatively low value. I guess maybe solving the problem of loss divergence then can solve the problem of reward.
this photo is a screenshot during training
![0](https://user-images.githubusercontent.com/36686362/130610432-78bab4cf-b5ed-4655-aeab-578273020b31.PNG)

**Comment by Trinkle23897** [comment]
see https://github.com/thu-ml/tianshou/issues/374#issuecomment-904508308 and https://github.com/thu-ml/tianshou/issues/400#issuecomment-901222278

**Comment by fentuoli** [comment]
> see [#374 (comment)](https://github.com/thu-ml/tianshou/issues/374#issuecomment-904508308) and [#400 (comment)](https://github.com/thu-ml/tianshou/issues/400#issuecomment-901222278)

Thanks for your reply, I try to replace 
_optim = torch.optim.Adam(
        list(actor.parameters())+(critic.parameters()), lr=lr)_    
with
 _optim = torch.optim.Adam(
        set(actor.parameters()).union(critic.parameters()), lr=lr)_
the loss is also becoming larger and larger.
Then I change _value_clip_ and _rew_norm_ from False to True, the loss is really large at first and after  about a dozen epochs it produced distribution errors.
![10](https://user-images.githubusercontent.com/36686362/130614825-c2dbc205-1577-4b3b-9cb4-35e55eafc581.PNG)

**Comment by Trinkle23897** [comment]
have you changed this line? https://github.com/thu-ml/tianshou/blob/d161059c3d6c7db1a6028ffbeb6fa5374ca8e381/tianshou/policy/modelfree/ppo.py#L142

**Comment by fentuoli** [comment]
> have you changed this line?
> 
> https://github.com/thu-ml/tianshou/blob/d161059c3d6c7db1a6028ffbeb6fa5374ca8e381/tianshou/policy/modelfree/ppo.py#L142

wait a moment

**Comment by fentuoli** [comment]
> > have you changed this line?
> > https://github.com/thu-ml/tianshou/blob/d161059c3d6c7db1a6028ffbeb6fa5374ca8e381/tianshou/policy/modelfree/ppo.py#L142
> 
> 
![3](https://user-images.githubusercontent.com/36686362/130618298-a9de56f5-4781-4803-9e0e-29294029e8ff.PNG)
actually, there is no change :(

**Comment by Trinkle23897** [comment]
send me an email again?

**Comment by fentuoli** [comment]
> send me an email again?

I don’t know what to say, thank you so much！
